assets/nginx.conf

```
@@ -59,7 +59,7 @@ http {
             # redirects, we set the Host: header above already.
             proxy_redirect off;
             proxy_pass http://pulp-api;
-            client_max_body_size 10m;
+            client_max_body_size 1024m;
```

maybe we should disable it
```suggestion
            client_max_body_size 0;
```
https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size

---

+1 rpm can very much very in size